nixos/modules/misc/ids.nix

```diff
@@ -297,6 +297,7 @@
       clickhouse = 278;
       rslsync = 279;
       minio = 280;
+      postage = 281;
```

Comment:
Static uids have a few legit use cases, for sure, but I'm not convinced by the predictability argument on its own. Enforcing ownership before use or at the transfer boundary seems much more robust, as it actually addresses the thing you care about.

How often will you be transferring data owned by this service between systems? Is it worth claiming a uid which can "never" be used again (if we're being literal about relying on static uids to preserve ownership in perpetuity) compared to the cost of `chown`'ing the data on first use? If it's very frequently transferred or changing ownership is very costly, then sure, but otherwise it seems wasteful/pointless.